src/rgw/rgw_rados.cc

```diff
@@ -4898,6 +4898,7 @@ static void accumulate_raw_stats(const rgw_bucket_dir_header& header,
     const rgw_bucket_category_stats& header_stats = pair.second;
 
     RGWStorageStats& s = stats[category];
+    s.set_dump_utilized(true);
```

Comment:
I'm confused--we don't seem to know here that there is a value for the utilized members

Reply:
this may be more clear if it defaults to true, and is disabled only at the sites where we're dumping it as user stats

Reply:
The idea of the old solution was that any time you tried to accumulate into the utilized value, you triggered it being displayed. I switched it around so the instance created for user stats sets the display to false, and by default it's true.